jsapp/js/account/usage.component.tsx

```diff
@@ -0,0 +1,115 @@
+import React, {useEffect, useState} from 'react';
+import styles from './usage.module.scss';
+import {formatMonth} from '../utils';
+import {getUsage} from './usage.api';
+import Icon from '../components/common/icon';
+import {NavLink} from 'react-router-dom';
+import Button from '../components/common/button';
+
+interface UsageState {
+  storage: number;
+  monthlySubmissions: number;
+  transcriptionMinutes: number;
+  translationChars: number;
+  showBanner: boolean;
+}
+
+export default function Usage() {
+  const [usage, setUsage] = useState<UsageState>({
+    storage: 0,
+    monthlySubmissions: 0,
+    transcriptionMinutes: 0,
+    translationChars: 0,
+    showBanner: true,
+  });
+
+  useEffect(() => {
+    getUsage().then((data) => {
+      setUsage({
+        ...usage,
+        storage: data.total_storage_bytes / 1000000, // bytes to GB
+        monthlySubmissions: data.total_submission_count_current_month,
+        transcriptionMinutes: data.total_nlp_asr_seconds / 60, // seconds to minutes
+        translationChars: data.total_nlp_mt_characters,
+      });
+    });
+  }, []);
+
+  function hideHeader() {
+    setUsage({
+      ...usage,
+      showBanner: false,
+    });
+  }
+
+  return (
+    <div className={styles.root}>
+      <h2>{t('Your account total use')}</h2>
+
+      {usage.showBanner && (
+        <div className={styles.header}>
+          <Icon name='alert' size='s' />
+          <div className={styles.article}>
+            <p>
+              {t(
+                'Please note these figures are only updated once per day. Numbers may not reflect immediately recent changes in account usage. For any questions concerning usage, please read the '
```

Comment:
I'm pretty sure that @JacquelineMorrissette told me this isn't true. I think I just made this up in my head one day.

Reply:
Can confirm, all our counters are updated in real time :+1: